src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -2817,29 +2819,25 @@ public boolean isRespawnAnchorWorks()
 	@Override
 	public boolean hasRaids()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return environment != Environment.NETHER;
```

Comment:
Does the end have raids?

Reply:
Havent looked at it ingame. Just wanted to replicate paper logic witch uses nms getter.
net.minecraft.data.worldgen.DimensionTypes
<img width="1117" alt="Bildschirmfoto 2023-10-11 um 19 19 27" src="https://github.com/MockBukkit/MockBukkit/assets/31861387/14d7d9f5-81d6-4459-a457-f6d55fab864d">

( line wraps were added for better readbilityy )